tests/test.py

```diff
@@ -59,7 +59,7 @@ def test_response_close():
     assert r.status_code == 200
     response = r.json()
     assert response['status'] == 'ok'
-    assert response['predictions'][0]['distance'] == str(0.0002380952380952381)
+    assert response['predictions'][0]['distance'] == 0.0002380952380952381
```

Comment:
For these kinds of tests, we probably want to use `pytest.approx` to a reasonable precision.